use-timescale/data-tiering/disabling-data-tiering.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Creating a tiering policy
+excerpt: How to create a tiering policy
+product: [ cloud ]
+keywords: [ tiered storage, tiering ]
+tags: [ storage, data management ]
+---
+
+# Disabling tiering on a hypertable
+
+If you no longer want to use tiered storage for a particular hypertable, you 
+can drop the associated metadata by calling the `disable_tiering` function.
+
+In order to use this API, first call [remove_tiering_policy][tiering-policy] to drop 
+any tiering policy associated with this hypertable. Second, make sure that there is 
+no tiered data associated with this hypertable.
+
+This query lists the tiered chunks associated with this hypertable.
+```sql
+select * from timescaledb_osm.tiered_chunks 
+```
```

Comment:
I checked a few doc pages, example queries are usually ended by `;` (surprisingly not all though, but most).

Also technically this query shows all tiered chunks, not just "associated with this hypertable". I wonder if this should be rephrased somehow or the query adjusted.

Reply:
fixed.